commands.json

```diff
@@ -4341,8 +4341,37 @@
         "type": "key"
       },
       {
-        "name": "ID",
-        "type": "string"
+        "type": "block",
+        "block": [
+          {
+            "command": "MAXLEN"
+          },
+          {
+            "type": "block",
+            "block": [
+              {
+                "command": "~",
```

Comment:
the text in t_stream.c says:
```
/* XADD key [MAXLEN [~|=] <count>] <ID or *> [field value] [field value] ... */
```
so i guess you need `=` in addition to `~`.

Reply:
LOL - I didn't know/remember this was actually supported, maybe it wasn't documented intentionally? I pushed a corrective PR (and updated the XADD screenshot + the commands' pages) but since it is a) the default b) undocumented (and maybe c) MAXLEN isn't likely to evolve much), perhaps we should revert this.

Reply:
i don't think we should revert this.
it seems good that the exact (default) behavior can be explicitly requested rather than implicitly.

but i now see it's not covered by any test test, so maybe it's not at all working.
do you mind creating a PR to add it to the tests? if it'll not work, we can consider trimming it from the docs, but if it does, i think we must keep supporting it, have it in the tests and docs.

Reply:
https://github.com/redis/redis/pull/8083